packages/ndla-ui/src/LicenseByline/EmbedByline.tsx

```diff
@@ -123,34 +122,20 @@ const EmbedByline = ({ type, topRounded, bottomRounded, description, children, v
     );
   }
 
-  const { copyright } = props;
-
-  const license = copyright ? getLicenseByAbbreviation(copyright.license?.license ?? "", i18n.language) : undefined;
-  const authors = getLicenseCredits(copyright);
-  const captionAuthors = Object.values(authors).find((i) => i.length > 0) ?? [];
-
-  const authorAndLicenseLink = () => {
-    return (
-      <>
-        {` ${t(`embed.type.${type}`)}${captionAuthors.length ? ": " : ""}`}
-        {captionAuthors.map((author) => author.name).join(", ")}
-        {license ? (
-          <>
-            {" / "}
-            <LicenseLink license={license} />
-          </>
-        ) : null}
-      </>
-    );
-  };
+  // const { copyright } = props;
 
   return (
     <BylineWrapper>
       <LicenseContainer>
         {description ? (
-          <LicenseDescription description={description}>{authorAndLicenseLink()}</LicenseDescription>
+          <LicenseDescription description={description}>
+            <LicenseAuthors type={type} copyright={props.copyright} />
+          </LicenseDescription>
         ) : (
-          <span>{authorAndLicenseLink()}</span>
+          <span>
+            {" "}
```

Comment:
Eg synes slike konstruksjoner ser litt rart ut. At en putter inn mellomrom for å justere designet. 
Ser i LicenseAuthors så returnerer du et mellomrom som første teikn i linje 28, så denne trengs vel uansett ikkje.

Reply:
Tenker du det samme om `{" / "}` i linje 32 i _LicenseAuthors_?

Reply:
Den er _litt_ bedre siden du faktisk har en verdi å putte inn.